db/migrations/000001_initial_schema.down.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+DROP TABLE users
```

Comment:
The tables deletion order should be in reverse order of the creation. 
For instance this script will fail as there is a reference between loans and users and you are trying to delete users first.